communications/journal.md

```diff
@@ -4,30 +4,28 @@ Journals don't always need to be like a ["Dear Diary" moment](https://gist.githu
 
 ## Objectives
 
-- Journals help you organize thoughts and recollect memory.
-- They give your teammates more context of where you have been.
-- The evolving plan component offers your teammate insights into the direction you're heading toward.
+- Helps to organize thoughts and recollect memory of daily tasks.
+- Provides teammates with more context of what each other has been working on & gain insights on project direction.
+- Feeds into weekly digests as a part of the meeting notes generation process.
 
 ## Guide
 
-### First Principle - Summarize Last Work Session
+### First Principle - Summarize Last Work Day
 
-The first and most important principle is to write down what you have done in the _Last Work Session_ as comprehensively as possible.
+The first and most important principle is to fill out the _Last Workday_ section as comprehensively as possible. Notes in this section allows others in the team to know what you have been doing, so that more efficient collaboration can be formed from this knowledge. This section is also being sourced into weekly digests we generate for meeting notes, so it is key to keep them clear and concise.
 
-We can be much better at our planning by reinforcing and associating back to what we have done.
-
-This also allows others in the team to know what you have been doing, so that more efficient collaboration can be formed from this knowledge.
+Our [avail_bot](https://github.com/EQWorks/avail-bot) routine for Dev Journal auto-fills _Last Workday_ with completed prev-day tasks, you could review & re-format these notes by directly typing into the Last Workday section.
 
 Adaptations:
 
-- Sessions can be in different lengths, such as every day, every other day, or every week.
-- However, it is better to have consistent session lengths. This allows members in collaboration with you to have a predictable period to check-in with you.
+- We have an automation tool to bring completed prev-day tasks to auto-fill this section.
+- Please edit this section if you find any missing prev-day tasks, or if prev-day tasks were poorly worded.
```

Comment:
seems redundant with the previous paragraph, I'd merge both and remove `adaptations`

Reply:
agree. the point-by-point presentation may sometime grab reading attention better. so perhaps adapt the previous paragraph into the points here, but revise the term and keep it consistent as _last workday_